### ISSUE TITLE
Set button(s) color?

### ISSUE THREAD

**Issue by rfranolla** [Platonenkov/Notification.Wpf#57]
Hey, 

I’ve been working on a project that utilizes the notification banner and the project allows users to set the color of the banner to anything they like.

I’ve notice that in some cases the button of the banner can become hard to see depending on the color set. For example, I set the banner to Red (#FF0000) the border of the button becomes hard to see (screenshot below). 
![Screenshot 2023-06-27 101028](https://github.com/Platonenkov/Notification.Wpf/assets/43393284/d5add788-5526-4c74-8156-c493445fc598)

Example code to create the banner:
```          
var notificationContent = new NotificationContent
{
       Title = "Example",
       Message = "Some Text,
       LeftButtonContent = "Dismiss",
       Background = Brushes.Red,
       Foreground = Brushes.Black
};
```

Would you consider allowing for the ability to set the color of the button(s), or the ability to specify how much of a ‘color difference’ the button(s) should be? Basically something to have the button stand out a bit more

Looking forward to knowing what you think!

**Comment by ICUBE321** [comment]
Yeah, I also noticed this and it gets really hard seeing the buttons using certain colors. Would be really great if this was addressed :) !

**Comment by Platonenkov** [comment]
fixed in v7.0.0.1
Thank you for your support @rfranolla
